Jenkinsfile

```diff
@@ -477,7 +479,70 @@ EOL
                 }
             }
         }
-	}
+        stage('Package helm charts'){
+            steps{
+                dir ("${WORKSPACE}/deploy/helms") {
+                    script {
+                        chartNames = sh(returnStdout: true, script: 'ls').split()
+                        println (chartNames)
+                        for (chart in chartNames){
+                            sh"""
+                            sed -i 's@^version: .*\$@version: ${Globals.buildVersion}@g' ${chart}/Chart.yaml
+                            helm package ${chart}
+                            """
+                        }
+                    }
+                }
+            }
+        }
+        stage('Deploy helm charts'){
+            steps{
+                withCredentials([[
+                 $class: 'UsernamePasswordMultiBinding',
+                 credentialsId: 'nexus-local-repository',
+                 usernameVariable: 'USERNAME',
+                 passwordVariable: 'PASSWORD']]) {
+                    dir ("${WORKSPACE}/deploy/helms") {
+                        script {
+                            for (chart in chartNames){
+                               sh"""
+                               curl -u ${USERNAME}:${PASSWORD} ${params.NexusHelmRepository} --upload-file ${chart}-${Globals.buildVersion}.tgz
+                               """
+                            }
+                        }
+                    }
+                }
+                dir ("${WORKSPACE}/legion-helm-charts") {
+                    script{
+                        if (params.StableRelease) {
+                            stage('Publish helm charts to Public repo'){
+                                if (params.UpdateVersionString){
```

Comment:
We should always set new version for release packages. UpdateVersionString parameter just says that version.py will be modified and uploaded to develop branch automatically.

Reply:
Removed unneeded steps.